tools/distrib/yapf_code.sh

```diff
@@ -53,7 +53,7 @@ for dir in $DIRS; do
   tempdir=`mktemp -d`
   cp -RT $dir $tempdir
   $PYTHON -m yapf -i -r -p $exclusion_args $dir
-  if ! diff -rq $dir $tempdir; then
+  if ! diff -r $dir $tempdir; then
```

Comment:
I did debate the former point with myself on the pylint_code.sh case, and eventually decided on a quick exit.  I like quick exits as, in principle, the more you stay in a test after a failure is detected, the greater the risk of a potential bug in the test script incorrectly masking the output, and since the test does not seem to generate a useful summary that shows specifically which diff command failed, the user will likely have to manually investigate after noticing a yapf test failure anyway. (Of course your point 2 was not applicable in the pylint case, as it does not generate temporary data, but cleaning up is possible in the branch).